src/assets/translations/en/main.json

```diff
@@ -632,12 +632,20 @@
   "trade": {
     "trade": "Trade",
     "allowance": "Allowance",
+    "slippageInfo": "Your transaction will revert if the price changes unfavorably by more than this percentage.",
     "allowanceTooltip": "The amount you will grant the smart contract permission to use on your behalf.",
     "searchingRate": "Fetching rate...",
     "noRateAvailable": "No rate available.",
     "previewTrade": "Preview Trade",
     "rate": "Rate",
     "tradeFrom": "Trade from",
+    "slippage": {
+      "maxSlippage": "Max Slippage",
+      "auto": "Auto",
+      "custom": "Custom",
+      "warning": "Your transaction may be frontrun and result in an unfavorable trade.",
```

Comment:
this warning should be more general, there's front running/sammich/bunch of ways MEV txs

maybe something like "High slippage tolerances can result in unfavorable trades."